javascript/javascript2/week1/main.js

```diff
@@ -0,0 +1,61 @@
+//Find the shortest word
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function shortestWord(compare) {
+  return compare.reduce((a, b) => (a.length <= b.length ? a : b));
+}
+shortestWord(danishWords);
+
+//Find and count the Danish letters
+//Find the individual number and the total number of Danish letters in a string.
+
+const danishLetters = ["æ", "å", "ø"];
+function find(str) {
+  let letters = { total: 0 };
+  for (let i = 0; i < str.length; i++) {
+    if (danishLetters.includes(str[i])) {
+      letters.total = letters.total + 1;
+      if (letters.hasOwnProperty(str[i])) {
+        letters[str[i]] = letters[str[i]] + 1;
```

Comment:
in here you could use += to update the value and increment it at the same time. 

`letters[str[i]] += 1`

Reply:
Thank you, I added the line